src/Core/Worker.php

```diff
@@ -0,0 +1,993 @@
+<?php
+namespace Friendica\Core;
+
+use Friendica\App;
+use Friendica\Core\System;
+use Friendica\Core\Config;
+use Friendica\Core\Worker;
+use Friendica\Util\Lock;
+
+use dba;
+use dbm;
+
+/**
+ * @file src/Core/Worker.php
+ *
+ * @brief Contains the class for all worker relevant stuff
```

Comment:
could you add a hint for what the workers are about? Could just be "for all worker (background process) relevant stuff" or so.

Reply:
Done